src/renderer/pages/Project.tsx

```diff
@@ -59,6 +73,14 @@ const ProjectPage = () => {
             videoResizeOptions
           ) => (
             <>
+              <CloudConfigView
```

Comment:
Rather than repeating yourself for each page can you put this in AppContents instead